File: recaudacion/2.Codigo/backend/RecaudacionApiGuiaSalidaBien/Domain/GuiaSalidaBien.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace RecaudacionApiGuiaSalidaBien.Domain
{
    [Table("GUIA_SALIDA_BIENES")]
    public class GuiaSalidaBien
    {
        [Key]
        [Column("GUIA_SALIDA_BIEN_ID")]
        public int GuiaSalidaBienId { get; set; }
        [Column("UNIDAD_EJECUTORA_ID")]
        public int UnidadEjecutoraId { get; set; }
        [Column("TIPO_DOCUMENTO_ID")]
        public int TipoDocumentoId { get; set; }
        [Column("GUIA_SALIDA_BIEN_NUMERO")]
        public string Numero { get; set; }
        [Column("GUIA_SALIDA_BIEN_FECHA_REGISTRO")]
        public DateTime FechaRegistro { get; set; }
        [Column("GUIA_SALIDA_BIEN_JUSTIFICACION")]
        public string Justificacion { get; set; }
        [Column("GUIA_SALIDA_BIEN_ESTADO")]
        public int Estado { get; set; }
        [NotMapped]
        public string EstadoNombre { get; set; }
        [Column("USUARIO_CREADOR")]
        public string UsuarioCreador { get; set; }
        [Column("FECHA_CREACION")]
        public DateTime FechaCreacion { get; set; }
        [Column("USUARIO_MODIFICADOR")]
        public string UsuarioModificador { get; set; }
        [Column("FECHA_MODIFICACION")]
        public DateTime? FechaModificacion { get; set; }
    }

}

File: recaudacion/2.Codigo/frontend/src/app/core/models/guiasalidabien.ts
export class GuiaSalidaBien {
    guiaSalidaBienId: number;
    unidadEjecutoraId: number;
    tipoDocumentoId:number;
    numero: string;
    fechaRegistro: any;
    justificacion: string;
    estado: number;
    estadoNombre: string;
    usuarioCreador: string;
    usuarioModificador: string;
    guiaSalidaBienDetalle: any;
    index:number;
}

export class GuiaSalidaBienEstado {
    guiaSalidaBienId: number;
    estado: number;
    usuarioModificador: string;
}

export class GuiaSalidaBienFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    unidadEjecutoraId: number;
    numero: number;
    fechaInicio: any;
    fechaFin: any;
    estado: number;
    rol: string;
}

export class GuiaSalidaBienDetalle {
    guiaSalidaBienDetalleId: number;
    guiaSalidaBienId: number;
    catalogoBienId: number;
    catalogoBien: any;
    ingresoPecosaDetalleId: number;
    ingresoPecosaDetalle:any;
    cantidad: number;
    serieFormato: string;
    serieDel: number;
    serieAl: number;
    estado:string;
    anioPecosa: number;
    numeroPecosa: number;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}
